PythonVisualizations/OrderedArray.py

```diff
@@ -0,0 +1,328 @@
+import random
+import time
+from tkinter import *
+try:
+    from drawable import *
+    from VisualizationApp import *
+except ModuleNotFoundError:
+    from .drawable import *
+    from .VisualizationApp import *
+
+CELL_SIZE = 50
+CELL_BORDER = 2
+CELL_BORDER_COLOR = 'black'
+ARRAY_X0 = 100
+ARRAY_Y0 = 100
+FONT_SIZE = 20
+VALUE_FONT = ('Helvetica', FONT_SIZE)
+VALUE_COLOR = 'black'
+FOUND_COLOR = 'brown4'
+
+class Array(VisualizationApp):
```

Comment:
The class name should be `OrderedArray`, so that it's not confused with the Array in Array.py